docs/README.md

```diff
@@ -119,6 +122,55 @@ Note:
 
 ---
 
+## Developing a LabVIEW measurement client
+
+1. Create and save a new LabVIEW project.
+
+2. From the project window, go to `Tools` → `Plug-In SDKs` → `Measurements` → `Create Measurement Plug-in Client...`.
+    - In the dialog, select atleast one of the available V2 measurements, then click `Create Measurement Plug-in Client(s)`.
+
+      ![New measurement client dialog](images/New%20measurement%20client%20dialog.png)
+
+    - This will create a new measurement client library in the project.
+
+      ![Measurement client library files](images/Measurement%20client%20library%20files.png)
+
+Note:
+
+- The Measurement Plug-In Client is compatible with all datatypes supported by the Measurement Plug-In.
+- For Enum datatypes, the LabVIEW Client supports only contiguous enums.
+- Ring controls in the Measurement Plug-In will be represented as numeric controls in the client, maintaining the same representation.
```

Comment:
You might want to clarify this a bit by saying if the measurement utilizes a LabVIEW user interface, any ring controls used in the UI will be represented as numeric controls by the generated client.